application/templates/admin/merge_data_sources.html

```diff
@@ -0,0 +1,57 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for("static_site.index"), "text": "Ethnicity facts and figures"},
+    {"url": url_for('admin.index'), "text": "Admin"}
+  ]
+%}
+
+{% block pageTitle %}Merge data sources{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-xl">Merge data sources</h1>
+
+      <div class="govuk-body">All of the following data sources will be merged together.</div>
+      <form>
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
+
+            <legend class="govuk-fieldset__legend">
+                Which one would you like to keep?
+            </legend>
+            {% for data_source in data_sources %}
```

Comment:
Can we use a WTForm for the fields here?

For a (WIP) example, see: https://github.com/racedisparityaudit/ethnicity-facts-and-figures-publisher/commit/94b522c517e42cfc54db57e22addce6a5af99a82